espnet/nets/pytorch_backend/transformer/plot.py

```diff
@@ -33,12 +33,12 @@ def _plot_and_save_attention(att_w, filename, xtokens=None, ytokens=None):
         ax.yaxis.set_major_locator(MaxNLocator(integer=True))
         # Labels for major ticks
         if xtokens is not None:
-            ax.set_xticks(numpy.linspace(0, len(xtokens) - 1, len(xtokens)))
-            ax.set_xticks(numpy.linspace(0, len(xtokens) - 1, 1), minor=True)
+            ax.set_xticks(numpy.linspace(0, len(xtokens), len(xtokens) + 1))
+            ax.set_xticks(numpy.linspace(0, len(xtokens), 1), minor=True)
```

Comment:
I met following error:

> ValueError: The number of FixedLocator locations (5), usually from a call to set_ticks, does not match the number of ticklabels (6).

`set_xticks` receive  invalid arguments with  the args of `set_xticklabels`: len(xtokens) != len(xtokens != [""])

I think `set_xticks` is, maybe, not required here, but I'm not sure. I simply changed the number to fix this problem.